mappings/net/minecraft/command/argument/IdentifierArgumentType.mapping

```diff
@@ -2,6 +2,9 @@ CLASS net/minecraft/class_2232 net/minecraft/command/argument/IdentifierArgument
 	FIELD field_21506 UNKNOWN_PREDICATE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD field_24267 UNKNOWN_ATTRIBUTE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD field_28002 UNKNOWN_ITEM_MODIFIER_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD field_36370 INVALID_BIOME_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD field_36371 INVALID_FEATURE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD field_36372 INVALID_STRUCTURE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```

Comment:
The translation keys here seem to be kept for compatibility with existing resource packs. Depending on the translations of the messages, these should be 'unknown' rather than 'invalid' for consistency:

```suggestion
	FIELD field_36370 UNKNOWN_BIOME_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
	FIELD field_36371 UNKNOWN_FEATURE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
	FIELD field_36372 UNKNOWN_STRUCTURE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```